src/viewport/viewport.js

```diff
@@ -55,17 +68,20 @@ export default class WebGLViewport extends Viewport {
   }
 
   getUniforms({
+    modelMatrix = null,
     projectionMode = COORDINATE_SYSTEM.LNGLAT,
-    positionOrigin = [0, 0],
-    modelMatrix = null
+    positionOrigin = [0, 0]
   } = {}) {
     // TODO: move the following line to initialization so that it's done only once
-    const positionOriginPixels = this.projectFlat(positionOrigin);
+    const positionOriginPixels = this.viewport.projectFlat(positionOrigin);
 
     const projectedPositionOrigin =
       [positionOriginPixels[0], positionOriginPixels[1], 0.0, 1.0];
-    const projections = this.getProjections();
+
+    const projections = this.viewport.getMatrices();
```

Comment:
> const projections = this.viewport.getMatrices({});

Fixing in `getMatrices` instead by adding default arg
@autobind getMatrices({modelMatrix = null, ...opts} **= {}**) {